### PR TITLE
[infra] Add support for rules_fuzzing's java_fuzz_test macro

### PR DESCRIPTION
**Pull request by fmeum**
@stefanbucur There probably won't be many multi-language Bazel projects on OSS-Fuzz other than `rules_fuzzing` itself, but I believe that something along these lines is the correct way to make CI Fuzz possible for `rules_fuzzing`. Could you take a look?

### CODE REVIEW

File: projects/bazel-rules-fuzzing-test-java/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN git clone -b java-oss-fuzz-2 https://github.com/fmeum/rules_fuzzing.git
```
Comment (stefanbucur) [inline projects/bazel-rules-fuzzing-test-java/Dockerfile:19]: I believe this would need to be changed to the main repo, right?

**Review by stefanbucur — COMMENTED** [review]
The changes look good to me. Leaving to @jonathanmetzman and @inferno-chromium for approval.

File: projects/bazel-rules-fuzzing-test-java/Dockerfile (hunk 1 above)
Comment (fmeum) [reply projects/bazel-rules-fuzzing-test-java/Dockerfile:19]: Yes, I will change it back to the main repo now that you have had a chance to try it out.

Comment (fmeum) [reply projects/bazel-rules-fuzzing-test-java/Dockerfile]: Done.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**